_stbt/core.py

```diff
@@ -1143,7 +1143,8 @@ def save_frame(image, filename):
     cv2.imwrite(filename, image)
 
 
-def wait_until(callable_, timeout_secs=10, interval_secs=0, stable_secs=0):
+def wait_until(callable_, timeout_secs=10, interval_secs=0, predicate=None,
+               stable_secs=0):
```

Comment:
Hmm now I remember why I put it in that order -- it makes more sense to document `predicate` first, since `predicate` can be documented without mentioning `stable_secs`, but `stable_secs` needs to talk about `predicate`. I think I'll leave it as is.